runserver.py

```diff
@@ -5,4 +5,4 @@
 logging.config.dictConfig(config.LOGGING_DICT)
 
 if __name__ == '__main__':
-    app.run(debug=config.DEBUG)
+    app.run(debug=config.DEBUG, host='0.0.0.0')
```

Comment:
can you find a way to do this only for the docker setup? I don't like it if I'm running on a host with a public ip